raiden/blockchain/state.py

```diff
@@ -79,3 +70,29 @@ def get_channel_state(
     )
 
     return channel
+
+
+def get_channel_state(
```

Comment:
why the indirection?

Reply:
I want to be able to test the logic of the function without mocking the `payment_channel_proxy`.

Reply:
ah, understood! 
for long term yield of the test, it might be beneficial to break this into smaller units, but I don't want to delay this here any further.
however, I want to ask you to change `do_get_channel_state` to something more telling, such as `create_channel_state_from_blockchain_data`